packages/core/src/SessionEventListener.ts

```diff
@@ -51,10 +82,17 @@ type TIMEOUT_SET_ARGS = {
   eventName: typeof EVENTS.TIMEOUT_SET;
   listener: (timeoutId: number) => void;
 };
+/**
+ * @deprecated Use NEW_TOKENS_ARGS instead
+ */
```

Comment:
Can you add the deprecation warning on the signatures using this event name? E.g. `events.on(eventName: NEW_REFRESH_TOKEN_ARGS.eventName, listener: NEW_REFRESH_TOKEN_ARGS.listener`. Some people will not be using the types directly, and get the typing from IDE hints directly, so the deprecation should apply even if the literals are used and not the exported type.